tasks/task1/task1_2.es6.js

```diff
@@ -0,0 +1,13 @@
+import fs from 'fs';
+import csv from 'csvtojson';
+import { pipeline } from 'stream';
+import errorHandler from './utils/errorHandler.es6';
+
+const csvFilePath = 'files/node_mentoring_t1_2_input_example.csv';
```

Comment:
Consider separating your csv and text files by placing them in different folders.
E.x create a folders "csv" and "text" under "files" folder.

Also, consider renaming your "tasks" folder into "src" and creating a "core" folder, where you could store your project constants, like BASE_CSV_PATH = "../files/csv".